setup.py

```diff
@@ -78,7 +79,6 @@
     "Operating System :: Microsoft :: Windows",
     "Programming Language :: Python",
     "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.8",
```

Comment:
does this change not work with python 3.8?

Reply:
nope, it works - I was working off the branch that removes support for Python3.8 earlier. but I reversed that when it was clear we're not deprecating 3.8 anytime soon